views/homepage.twig

```diff
@@ -27,7 +27,7 @@
 
                     {#<br><h3>{{ month|date('F Y') }}</h3>#}
 
-                    {% for talk in talks %}
+                    {% for key, talk in talks %}
```

Comment:
OK, yeah we'll need to work on getting the talk title, speaker name + date into the URL for SEO++